src/solver/impls/cvode/cvode.cxx

```diff
@@ -458,6 +467,10 @@ int CvodeSolver::init() {
 #endif
   }
 
+  // Set internal tolerance factors
+  CVodeSetNonlinConvCoef(cvode_mem, cvode_nonlinear_convergence_coef);
+  CVodeSetEpsLin(cvode_mem, cvode_linear_convergence_coef);
```

Comment:
Ah, this is because we use the system version of SUNDIALS for the clang-tidy action, which turns out to be 3.1.2.

I think just updating `clang-tidy-review` will use a more recent version of Ubuntu and give us SUNDIALS 5.1.8, but I guess the question which versions of SUNDIALS do we want to support?